pub-critic/src/app/components/footer/Footer.module.scss

```diff
@@ -0,0 +1,75 @@
+.Footer {
+    width: 100%;
+    height: 200px;
+    background-color: black;
+    display: flex;
+    padding: 40px 5vw;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    @media (max-width: 768px) {
+        height: 300px;
+    }
+    .FooterContent {
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        width: 30%;
+        @media (max-width: 768px) {
+            width: 70%;
+        }
+        .SocialRow {
+            display: flex;
+            flex-direction: row;
+            justify-content: space-between;
+            align-items: center;
+            width: 100%;
+            margin-bottom: 20px;
+            img {
+                width: 2vw;
+                height: 2vw;
```

Comment:
would be great to have some converter function: 

```
$vw-base: 1440px;

@function strip-unit($num) {
  @if $num == auto {
    @return $num;
  }

  @return $num / ($num * 0 + 1);
}

@function convert-to-vw($value, $base-value: $vw-base) {
  @if $value == auto {
    @return $value;
  }

  $value: strip-unit($value) / strip-unit($base-value) * 100vw;

  @if ($value == 0) {
    $value: 0;
  } // Turn 0vw into 0

  @return $value;
}

@function vw-calc($values, $base-value: $vw-base) {
  $max: length($values);

  @if $max == 1 {
    @return convert-to-vw(nth($values, 1), $base-value);
  }

  $vwValues: ();

  @for $i from 1 through $max {
    $vwValues: append($vwValues, convert-to-vw(nth($values, $i), $base-value));
  }

  @return $vwValues;
}
```
 
` height: vw-calc(2);`

Reply:
I put it into the code, however it does not seem to work that well, I put auto instead and it seems fine, will ask this again today